superset-frontend/src/components/EmptyState/index.tsx

```diff
@@ -131,6 +131,11 @@ const ImageContainer = ({ image, size }: ImageContainerProps) => (
   />
 );
 
+const handleMouseDown = (e: SyntheticEvent) => {
```

Comment:
Why do we need that?

Reply:
Actually it is to solve the conflict between blur and click. Currently some of our search filters will trigger the query when blur, since blur will be triggered before click, resulting in click is not work (may also have this problem before). Calling preventDefault after the mousedown trigger will disable the blur so that the click can work properly.

Reply:
Ok, thanks for explanation!